tests/conftest.py

```diff
@@ -211,3 +250,13 @@ def test_result():
     }
 
     return test_res
+
+@pytest.fixture
+def token():
+    """Get the IBMQX token from an environment variable."""
+    t = os.getenv("IBMQX_TOKEN_TEST", None)
```

Comment:
@antalszava I named this `IBMQX_TOKEN` in the repository secret!

Reply:
or is the plan not to use this for CI anyway?

Reply:
Oh was wondering about it, thanks! Updated the names to use `IBMQX_TOKEN`.